lib/common/template.js

```diff
@@ -108,5 +108,18 @@ function templateServiceFactory(FileLoader, Constants, Promise, _, waterline, Lo
         });
     };
 
+    TemplateService.prototype.unlink = function (name, scope) {
```

Comment:
yep, that's why.  The unlink here is part of the unlink of the actual file during sku pack deletion.